neo/Network/P2P/TaskManager.cs

```diff
@@ -266,5 +271,13 @@ internal protected override bool IsHighPriority(object message)
                     return false;
             }
         }
+
+        internal protected override bool ShallDrop(object message, IEnumerable queue)
+        {
+            if (!(message is TaskManager.NewTasks tasks)) return false;
+            // Remove duplicate tasks
+            if (queue.OfType<TaskManager.NewTasks>().Any(x => x.Payload.Type == tasks.Payload.Type && x.Payload.Hashes.SequenceEqual(tasks.Payload.Hashes))) return true;
```

Comment:
Can you break this line or make this more readable in some other form? Thanks

Reply:
The change is outdated, please check PR #1024 for 2x.

Reply:
I will also check there @yongjiema, sorry for the delay for looking into this.

Lets make it live again.

Reply:
@yongjiema, let's just keep the ping with `LastBlockHeight` and try merge this PR asap.
The `PendingHashes` could be moved to a new PR and, then, we discuss it a little further.